src/frontend/screens/ObservationDetails/index.tsx

```diff
@@ -35,10 +37,24 @@ const ObservationDetails = ({
   route,
 }: NativeRootNavigationProps<"ObservationDetails">) => {
   const [{ preset }] = useDraftObservation();
-  const current: number = +route.params.question;
```

Comment:
I beleive this was a typo that made it into the code. Adding the current question number to itself does not make sense. the user is navigated to a NEW instance of this screen every time a new question appears, with the question param being updated.

Reply:
i don't think this is a typo actually. it's a (intentionally?) clever way of converting a potential string into a number 😅 e.g. `+"1"` results in `1`. probably not necessary given how we have react native navigation set up though

Reply:
ahh yes very clever. but yes, you are correct. With the new navigation we can pass all primitive types, and they will be extracted as the correct type